app/views/mirador/_show.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if mirador_displayable?(document) %>
+  <% query_param = @current_query ? "&q=#{url_encode(@current_query)}" : '' %>
```

Comment:
@mohideen So apparently I was wrong about needing to manually encode the query, since I just tried it with a query with a space, and it ended up double-encoding it and causing pcdm-manifests to throw a 500 error. If you reverse the latest commit, I can go ahead and merge it as it was initially in the PR.